\documentclass[a4paper,11pt]{article}
%\usepackage[cm]{fullpage}
\usepackage[margin=1.4cm]{geometry}
\usepackage{tikz}
\usetikzlibrary{shapes.multipart}

\usepackage[numbers]{natbib}
\usepackage{xcolor}
\usepackage{hyperref}
\usepackage{bbding}
\usepackage{pifont}
\usepackage{wasysym}
\usepackage{amssymb}
\usepackage{array}
\usepackage{longtable}
\newcommand{\red}[1]{{\color{red!80!orange}#1}}
\newcommand{\blue}[1]{{\color{blue!80!cyan}#1}}
\newcommand{\green}[1]{{\color{green!50!black}#1}}
\newcommand\ytl[2]{
\parbox[b]{8em}{\hfill{\color{cyan}\bfseries\sffamily #1}~$\cdots\cdots$~}\makebox[0pt][c]{$\bullet$}\vrule\quad \parbox[c]{4.5cm}{\vspace{7pt}\color{red!40!black!80}\raggedright\sffamily #2.\\[7pt]}\\[-3pt]}
\title{Timeline of Deliverables}
\author{Saket Choudhary}
\newcommand{\cmark}{\ding{51}}%
\newcommand{\xmark}{\ding{55}}%
\newcolumntype{P}[1]{>{\centering\arraybackslash}p{#1}}

\begin{document}
\maketitle

\textbf{Proposed title:} {Locating Transcription Factor Binding Sites: Methods in search of \textit{moralistic} motifs} 

\textbf{Expected length:} 8-10 pages

\textbf{References: } See References section. In places where required I have mentioned relevant review articles that I would be keeping as my main reference.

\textbf{Key reference}: Assessing computational tools for the
discovery of transcription factor binding sites \cite{tompa_assessing_2005}

\textbf{Expected duration}: 6 weeks($14^{th}$ March to $30^{th}$ April)

\textbf{Section Organization}: See table \ref{tab:table2}

\textbf{Progress tracking}: \url{https://trello.com/b/8hLZBkKA/review-paper} 

\newpage

%\begin{table}
\footnotesize
%\begin{tabular}{ P{3cm} p{4.6cm} p{4.6cm} P{2cm} P{2cm}  }
\begin{longtable}{ P{3cm} p{4.6cm} p{4.6cm} P{2cm} P{2cm}  }
\hline
\textbf{Tool} & \textbf{Original Findings} & \textbf{Principle} & \textbf{Reference} & \textit{de-novo} \\
\hline\\
AlignACE & Discovery of over represented motifs in unaligned sequences, typically in the upstream region of corregulated genes & Gibbs sampling & \cite{roth_finding_1998} & \cmark \\
\hline\\
ANN-Spec & finding \textit{low-complexity} patterns present in high frequency. Suitable for locating TFBS given a background sequence & Artificial Neural Network for parameter fitting to maximize posterior probability and then Gibbs sampling for & \cite{workman_ann-spec:_2000}  & \cmark\\
\hline\\
\red{Consensus} & Statistically significant alignments of DNA or protein sequences to determine evolutionary/functional perspective  & Greedy algorithm that searches for motifs maximizing information iteratively(Fixed width) & \cite{hertz_identifying_1999} & \xmark (Check, constraint: Fixed width)\\
\hline \\
\red{Consite} & TFBS prediction using phylogenetic footprinting & Accounts for evolutionary constraints by aligning regulatory sequence from orthologous pairs of genes  & \cite{sandelin_consite:_2004}   & \xmark\\
\hline \\
GLAM & Locating functional sites by MSA & Simulated Annealing for automatically determining the width(An improvement over Stormo's Gibbs sampling approach \cite{hertz_identifying_1999} & \cite{frith_finding_2004} & \cmark \\
\hline \\
The Improbizer & Identifying cis-regulatory elements that activate gene-expression within pharyngeal gene clusters & EM algorithm, zero, first or second order markov model for background sequences  & \cite{ao_environmentally_2004} & \cmark \\
\hline \\
MEME & Over-represented motifs in DNA, protein sequences  & EM algorithm to search for optimum motif & \cite{bailey_fitting_1994} & \cmark \\
\hline \\
\red{MONKEY} & Identifying conserved transcription factor binding sites in MSA & Probabilistic model of binding site-specificity accounting for evolutionary tree by modeling one as background  & \cite{moses_monkey:_2004} &  \xmark \\
\hline\\
\red{CENTIPEDE} & Used to predict genome wide map of 800K TFBS of 200+ TFs & Integrates histone modification, gene annotation, DNAse I cleavage pattern to predict TFBS & \cite{pique-regi_accurate_2011} & \cmark \\
\hline\\
SeqGL &  TFBS prediction using ChIP, DNAse  or ATAC-seq data & Group lasso regularization to extract most important \textit{k-mer} groups distinguishing peaks from flanking sequences followed by motif finding across regions that have non zero weight &  \cite{setty_seqgl_2015} & \cmark \\
\hline\\
YMF & Statistically significant motifs &  Given regulatory regions of \textit{related} genes, find motifs with greater Z-score & \cite{sinha_ymf:_2003} &  \xmark \\
\hline\\
Weeder & Predicting Regulatory motifs & Models the significant occurrence of motifs over a seventh order markov chain expected background & \cite{pavesi_weeder_2004} & \cmark \\
\hline\\
\red{TFEM} & TFBS prediction & Position specific priors based on phylogenetic conservation, penalization based on deviation from conserved profiles & \cite{kechris_detecting_2004} & \cmark \\
\hline\\
\red{Kellis et al.} & Prediction of regulator target for drosophila &  BLS: Branch Length score based cutoffs for finding most significant motifs  & \cite{kheradpour_reliable_2007} & \cmark \\
\hline\\
\red{PhyloGibbs} & Regulatory motif finding in multiple local sequence alignment of orthologous sequences  &  MCMC, simulated annealing based approach that treats alignments as the sites for binding and intergenic DNA as 'background', taking into account evolutionary distances& \cite{siddharthan_phylogibbs:_2005} & \xmark \\
\hline\\
REDUCE & Discovering cis-regulatory sequences using expression data without the need of gene clustering &  Models the log fold change expression as a linear with the number of occurrences(or Information content) of motif as covariates. Motif lengths are determined by user &  \cite{bussemaker_regulatory_2001} & \xmark \\
\hline \\
GMEP & Modeling Sequence to expression(S2E) profiles. Hierarchical clustering of GMEP identified clusters of motifs with known TFs & Enumerate motifs for different length to find the weight contribution to gene expression, similar to the REDUCE algorithm discussed above & \cite{chiang_visualizing_2001} & \xmark \\
\hline \\
\red{EMnEM/PhyME}  & Identify motifs in orthologous sequences & Phylogenetic EM based approach & \cite{sinha_phyme:_2004,moses_phylogenetic_2004} & \xmark\\
\hline \\
RCADE  & Motif discovery in C2H2-ZF ChIP-seq data & Use a previously established recognition code for C2H2 to identify  motifs in target sequences, which are then tested for enrichment using sequences from endogenous retroelements(ERE) and non-ERE regions &  \cite{najafabadi_identification_2015} & \xmark \\
\hline\\
%PhastCons & & & &\\
%\hline \\
DME & Identifying tissue specific TFBS & Identifies motifs over-represented in one set of sequences over the background(promoters of deferentially expressed genes) & \cite{smith_identifying_2005} & \cmark \\
\hline \\
\red{INSIGHT} & Not for motif discovery, but to gauge impact of mutations on TFBS& Probabilistic model to measure impact of natural selection on TFBS  & \cite{siepel_cis-regulatory_2014} & \xmark \\
\hline\\
%\red{ConSite} & Finding cis-regulatory sequences & Integration of TF specificity models with phylogenetic footprinting & \cite{sandelin_consite:_2004} & \xmark \\
%\hline\\
\red{rVISTA} & Finding cis-regulatory sequences & Clustering of TFBS and interspecies sequence conservation  & \cite{loots_rvista_2002} & \xmark \\
\hline\\
%GERP & & & & \\
%\hline\\
TRAWLER & Regulatory motif discovery pipeline & Uses a suffix-tree implementation and a Z-score approximation &\cite{ettwiller_trawler:_2007} & \cmark \\
\hline\\
\red{MEME-prior} & TFBS prediction & Prior probabilities based on phylogenetic or other background information assigned to bases & \cite{cuellar-partida_epigenetic_2012} & \cmark \\
\hline\\
\red{FootPrinter} & Regulatory motif prediction in homologous sequences & Uses MSA and evolutionary conservation to determine motifs & \cite{blanchette_footprinter:_2003} & \xmark \\
\hline\\
\caption{Tools, purpose, methods}
\label{tab:table1}
\end{longtable}

\begin{table}
\caption{Timeline of deliverables}
\label{tab:table2}
\centering
\footnotesize
\begin{minipage}[t]{.7\linewidth}
\color{gray}
\rule{\linewidth}{1pt}
\bigskip
\ytl{$14^{th}$ March}{Deliverables proposed}
\bigskip
\ytl{$22^{nd}$ March}{Page 1-2: Introduction:\footnotesize\begin{itemize}\itemsep0em 
\item Why TFBSs matter? : Biological motivation (Relevant review: \cite{wray_evolutionary_2007}
\item TFBS prediction : Computational Challenges(Hints from review article \cite{weirauch_evaluation_2013})
%\item Non consensus of numerous methods: Implications(Different methods
%suit different experiments: Hints from review article \cite{weirauch_evaluation_2013})
%\item Section organization
\end{itemize}}
\bigskip
\ytl{$1^{st}$ April}{Page 3-4:TFBS discovery methods overview: A two step approach: \footnotesize\begin{enumerate}\itemsep0em 
\item Representation of Motif(Consensus Based or Profile Based) [Comprehensive Review: \cite{stormo_modeling_2013}]
%\begin{itemize}
%\item Position Weight Matrices (PWMs) \cite{stormo_dna_2000}, with gaps and %correlation \cite{hertz_identification_1990,hertz_identifying_1999}
%\item Dinucleotide Weight Matrices(DWMs) \cite{siddharthan_dinucleotide_2010}
%\item Other Variable width permitting models %\cite{sandelin_prediction_2005,lyakhov_discovery_2008,riley_p53hmm_2009}
%\item Regression Based models \cite{wang_interaction-dependent_2007,annala_linear_2011}
%\item Energy Modeling \cite{zhao_inferring_2009}
%\end{itemize}
\item Identifying binding sites given a motif representation
\end{enumerate}}
\bigskip
\ytl{$15^{th}$ April}{Page: 6-10 TFBS discovery methods:
\begin{itemize}
\item Comparative approaches which do not account for conservation information, See table \ref{tab:table1} (Just an overview)
\item Motif discovery and Phylogenetic footprinting various approaches, the relevant math involved, their shortcomings and biological relevance/generalization(Major chunk of discussion):
\begin{itemize}
\item INSIGHT
\item EMnEM/PhyME
\item CENTIPEDE
\item PhyloGibbs
\item MONKEY
\item Kellis et al.
\item TFEM
\item Consensus 
\item Consite
\end{itemize}
\end{itemize}}

%\item Motif discovery and Phylogenetic footprinting \begin{itemize}
%\item See table \ref{tab:table1}
%\end{itemize}
%\end{itemize}

\end{minipage}
\end{table}

\begin{table}
\centering
\footnotesize
\begin{minipage}[t]{.7\linewidth}
\ytl{$22^{nd}$ April}{Discussion: \begin{itemize}
\item Recommendations for what method to choose for what type of data that capture more information biologically
\item Recommendation for best practices? (steps to follow to minimize false positive motifs)
\item Overview of statistics used for assessing motif(tool performance) quality
\end{itemize}}
\bigskip
\ytl{$29^{th}$ April}{Conclusion}
\rule{\linewidth}{1pt}%
\end{minipage}%
\end{table}

\bibliographystyle{unsrt}
\bibliography{Zotero}
\end{document}